File: Secao06 - Estruturas de Decisão/Exercicios em Java/Exercicio03.java
package br.com.udemy.secao06;

import java.util.Scanner;

public class Exercicio03 {
	
	public static void main(String[] args) {
		int num, par, impar;
		Scanner teclado = new Scanner(System.in);
		
		System.out.print("Informe um numero inteiro para verificação: ");
		num = teclado.nextInt();
		
		if(num % 2 == 0) {
			par = num;
			System.out.println(par + " = numero par.");
		}else{
			impar = num;
			System.out.println(impar + " = numero impar.");
		}
		teclado.close();
	}

}

File: Secao06 - Estruturas de Decisão/Exercicios em Python/Exercicio03.py
# -*- coding: utf-8 -*-
print("Par ou Impar.\n")
num1 = int(input("Informe um numero para analise: "))

if num1 % 2 == 0:
    par = num1
    print("Numero {0} é Par.".format(par))
else:
    impar = num1
    print("Numero {0} é Impar.".format(impar))